app/controllers/editorial/submissions_controller.rb

```diff
@@ -10,7 +10,7 @@ class Editorial::SubmissionsController < Editorial::SectionsController
 
 
     def index
-      @submissions = scope.open.for(current_user)
+      @submissions = scope.open
```

Comment:
No you're right - I think it was only restricted by user. If we're to expose all submissions though, there isn't much a user can do with any but their own. Perhaps we have a lack of clarity around the intent of this page.